docs/products/redis/howto/estimate-max-number-of-connections.rst

```diff
@@ -0,0 +1,23 @@
+Estimate maximum number of connection in Aiven for Redis
+========================================================
+
+The number of simultaneous connections in Aiven for Redis depends on the total available memory on the server. 
+
+
+Here is how you can estimate it:
+
+.. math::
+
+   4\times m
+
+where `m` represents the memory in megabytes (MB) binary. With at least 10000 connections available, even on the smallest servers. For example, on a server with 4GB memory (4,096 MB binary), the simultaneous connections are:
```

Comment:
Do we need to specify "binary"? I believe saying memory in megabytes is self explanatory

Reply:
I would say to keep the binary as it could also be decimal, right? Done here: https://github.com/aiven/devportal/pull/423/commits/18b987e7dde55430c54200ce439806f5191d3bbe

Reply:
I would put ``m`` in double ` to show it as formula

Reply:
sorry, I understood wrong here @ftisiot you can check now!